mirrord/layer/src/tcp_steal.rs

```diff
@@ -102,8 +109,14 @@ impl TcpHandler for TcpStealHandler {
             .ok_or(LayerError::ListenAlreadyExists)?;
 
         tx.send(ClientMessage::TcpSteal(LayerTcpSteal::PortSubscribe(port)))
```

Comment:
Thus should be if else case where if no filter steal all and if yes filter steal with filter